test/parallel/test-cluster-dgram-1.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
+var common = require('../common');
```

Comment:
In cases like this, where you're moving the `require('../common')`, it might make sense to also move to `const`.